lib/buttercms/data_store_adapters/redis.rb

```diff
@@ -1,9 +1,16 @@
+begin
+  require 'redis'
+rescue LoadError
+  puts "WARNING: redis >= 3.0.0 is required to use the redis data store."
+  raise
+end
```

Comment:
I think it's fine for it to be. I was imagining that the LoadError's normal message is useful, and that printing the warning above it would just help clarify where it came from, so sort of an annotation on the load error. But raising a runtime error specific to this case is fine as well.